Branch-SDK/src/io/branch/referral/ServerRequestRegisterInstall.java

```diff
@@ -94,6 +95,7 @@ public void onRequestSucceeded(ServerResponse resp, Branch branch) {
             prefHelper_.setExternalIntentExtra(PrefHelper.NO_STRING_VALUE);
             prefHelper_.setAppLink(PrefHelper.NO_STRING_VALUE);
             prefHelper_.setPushIdentifier(PrefHelper.NO_STRING_VALUE);
+            prefHelper_.setIsAppLinkTriggeredInit(false);
```

Comment:
The primary pathway this can be triggered on on install. We'll need to pass the boolean up here.